docker-compose.yml

```diff
@@ -357,8 +357,6 @@ services:
     depends_on:
       pg_sql:
         condition: service_healthy
-      mgmt_backend:
```

Comment:
Is there any specific reason for removing this?

Reply:
@donch1989 
because when I was developing mgmt, I ran the make PROFILE=exclude-mgmt latest which make the artifact not able to launch leading docker-compose up fails. I know this is a dep for backend like pipeline and model but when I chech them. I don't see mgmt setup over them. So, I guess it is a trade off when we were in the case of developing the mgmt and wanna launch other service successfully.

Reply:
I suggest we keep this, given that artifact do depend on mgmt. For development, I usually don't use `PROFILE=exclude-xxx`. Instead, I use `PROFILE=all`, then `docker rm -f` the service I want to develop, and finally run `make dev` for that service.

Reply:
I see. Let's do it! I have added it back.